README.md

```diff
@@ -159,46 +72,84 @@ The summary details will be accessible in `$RUNNER_TEMP/build-scan-data-maven/bu
+The extension is automatically registered by configuring the environment `MAVEN_OPTS=-Dmaven.ext.classpath=<PATH_TO_EXTENSION>`.
+
+_Note:_<br>
+If `MAVEN_OPTS` environment variable is set in the step invoking the `mvn` command, the extension won't be registered.
+Make sure to use `MAVEN_OPTS: ${{ env.MAVEN_OPTS }} <EXTRA_PARAMETERS>` construction to append the extra parameters and have the extension registered.
+
+The captured files are added as workflow artifact (one artifact per job).
 
 **Event Triggers**:
 
-- `workflow_run`: to run after the build workflow. This event allows access to the repository secrets (_Develocity Access Key_) which is required to publish a Build Scan® to Develocity when authentication is enabled.
+- `pull_request`: To capture unpublished Build Scan®
+- `workflow_run`: To capture Build Scan® links
 
 **Permissions**:
 
 The following permissions are required for this action to operate:
 - `pull-requests: write`: to comment the pull-request
-- `actions: write`: to delete a workflow artifact
 
 **Action inputs**:
 
-| Name                             | Description                                                                  | Default               |
-|----------------------------------|------------------------------------------------------------------------------|-----------------------|
-| `develocity-url`                 | Develocity URL                                                               |                       |
-| `develocity-access-key`          | *Optional*: Develocity access key                                            |                       |
-| `develocity-allow-untrusted`     | *Optional*: Develocity allow-untrusted flag                                  | `false`               |
-| `skip-pr-comment`                | *Optional*: Whether to comment or not the pull-request with Build Scan links | `false`               |
-| `skip-job-summary`               | *Optional*: Whether to add or not a summary to the GitHub workflow           | `false`               |
-| `skip-project-id-in-job-summary` | *Optional*: Whether to add or not project id to the job summary / comment    | `true`                |  
-| `authorized-users-list`          | *Optional*: CSV List of users allowed to publish Build Scans                 | `''`                  |
-| `github-token`                   | *Optional*: Github token                                                     | `${{ github.token }}` |
+| Name                              | Description                                                                  | Default  |
+|-----------------------------------|------------------------------------------------------------------------------|----------|
+| `capture-strategy`                | *Optional*: Build Scan capture strategy (ALWAYS, ON_FAILURE, ON_DEMAND)      | `ALWAYS` |
+| `capture-unpublished-build-scans` | *Optional*: Whether to enable or not unpublished Build Scans capture         | `true`   |
```

Comment:
Ok so two thoughts here.
1. This (and the others) should say `Whether or not to enable unpublished scans capture`

2. Can we just get rid of all the `whether or not` and instead say `Enable Build Scan links capture` to make it more concise?

Reply:
Thanks, will apply suggestion 2., it is way more readable 👍